api/filter/http/ip_tagging.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter.http;
+
+// This is an HTTP filter which enables Envoy to tag requests with extra
+// information such as location, cloud source, and any extra data. This is
+// useful to prevent against DDoS.
+message IPTagging {
+  enum RequestType {
+    INTERNAL = 0;
+    EXTERNAL = 1;
+    BOTH = 2;
```

Comment:
Do you want to default internal, external or both? 0 is the default.

Reply:
Also no idea, from the docs - it is a string in docs.

Reply:
defaults to both.. fixed.